rs/embedders/fuzz/src/ic_wasm.rs

```diff
@@ -21,7 +21,7 @@ use wasmparser::*;
 lazy_static! {
     static ref UNIVERSAL_CANISTER_WASM_BYTES : BinaryEncodedWasm = {
         // Universal canister wasm is gzipped
-        decode_wasm(EmbeddersConfig::new().wasm_max_size, Arc::new(UNIVERSAL_CANISTER_WASM.to_vec())).expect("Unable to decode universal canister wasm")
+        decode_wasm(EmbeddersConfig::new().wasm_max_size, Arc::new(get_universal_canister_wasm())).expect("Unable to decode universal canister wasm")
```

Comment:
out of curiosity, why not use the (lazy) static value here?

Reply:
The docs [mention](https://docs.rs/lazy_static/latest/lazy_static/) this could leak to deadlock so I didn't want to run that risk here.